hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/volume/VolumeUsage.java

```diff
@@ -76,15 +76,18 @@ private long getRemainingReserved() {
     return Math.max(reservedInBytes - getOtherUsed(), 0L);
   }
 
-  public synchronized void start() {
```

Comment:
I think we should keep `start` and `shutdown` synchronized.  They may be called in various threads.  Any specific reason to remove it now?